Documentation/css/grid.css

```diff
@@ -0,0 +1,16 @@
+@import url("../../styles.css");
+@import url("../../stylesheets/navbar.css");
+@import url("../../stylesheets/footer.css");
+@import url("../../stylesheets/sidebar.css");
+@import url("./getstarted.css");
+@import url("../../components/grid/grid.css");
+
+.example {
+  width: 100%;
+  margin: 1rem 0;
+  display: flex;
+  flex-direction: column;
+  padding: var(--size-5);
```

Comment:
Does this variable follow the rem convention or px convention?
- If it's rem then pls change 1 rem to variable.
- If it's px then pls change 10px to variable.

Reply:
Variable is defined in root with rem units. Yes I should have used it on line 10 as well. Thanks for pointing it out!